Vizzuality/landgriffon
# Landgrif 1467 create prototype for exploring potential calculation of the nutrient assimilation capacity indicator

### General description

This PR includes:

- The preprocessing of the limiting nutrient dataset to obtain the percentage of reduction required to calculate the nutrient assimilation capacity indicator.
- The pipeline for ingesting both, the nutrient assimilation capacity needed for the indicator calculation and the limiting nutrient to be visualized as contextual data.
- Metadata for the two new datasets.

### Testing instructions

Make sure that the preprocessing work ok and ingest the new data in the database. Make sure that all the relations exist. 

## Checklist before merging
